src/index.js

```diff
@@ -184,8 +185,10 @@ class CreditCardInput extends Component<Props, State> {
   };
 
   handleCardExpiryBlur = (e: SyntheticInputEvent<*>) => {
-    if (!payment.fns.validateCardExpiry(e.target.value)) {
-      this.setFieldInvalid('Expiry date is invalid');
+    const cardExpiry = e.target.value.split(' / ').join('/');
+    const expiryError = creditCardExpiryValidator.isExpiryInvalid(cardExpiry)
```

Comment:
Here, it could also be an one-liner, like:

`const expiryError = creditCardExpiryValidator.isExpiryInvalid(e.target.value.split(' / ').join('/'))`, but I prefer keeping this in two different lines/declarations, so that the purpose of each one is more clear and the code is self-explained.

Reply:
Yeah, I'd rather it be readable.